package/MDAnalysis/analysis/density.py

```diff
@@ -595,6 +662,32 @@ def density_from_Universe(universe, delta=1.0, atomselection='name OH2',
     (Using the special case for the bulk with `soluteselection` and `cutoff`
     improves performance over the simple `update_selection` approach.)
 
```

Comment:
@orbeckst Unfortunately I am not sure what you mean by this comment. Since there is already an Examples heading on line 639, do you mean that there should be a subheading for each type of example?

Reply:
I didn't see it. Ok then.